### PR TITLE
Extract simple name tag system from cloudnet original cloud

### PR DESCRIPTION
**Pull request by TheMeinerLP**
I simplified the simple name tag api.
And do a standalone tool.

### CODE REVIEW

**Comment by TheMeinerLP** [comment]
I think is ready to merge. I have reviewed by my self again.

File: SimpleNameTagPlugin/src/main/resources/plugin.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: SimpleNameTag
+main: eu.cloudnetservice.simplenametag.SimpleNameTagPlugin
+version: ${nametag.version}
+
+api-version: 1.13
+author: TheMeinerLP
```
Comment (derklaro) [inline SimpleNameTagPlugin/src/main/resources/plugin.yml]: Maybe use CloudNETService here

File: SimpleNameTagPlugin/src/main/java/eu/cloudnetservice/simplenametag/SimpleNameTagSystem.java [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+            setColor.setAccessible(true);
+            if (permissionGroup.getColor().length() != 0) {
+                setColor.invoke(team, ChatColor.getByChar(permissionGroup.getColor().replaceAll("&", "").replaceAll("§", "")));
+            } else {
+                setColor.invoke(team, ChatColor.getByChar(ChatColor.getLastColors(permissionGroup.getPrefix().replace('&', '§'))
+                        .replaceAll("&", "")
+                        .replaceAll("§", "")));
+            }
+        } catch (NoSuchMethodException ignored) {
+        } catch (IllegalAccessException | InvocationTargetException e) {
```
Comment (derklaro) [inline SimpleNameTagPlugin/src/main/java/eu/cloudnetservice/simplenametag/SimpleNameTagSystem.java:155]: Rename e to exception (for example)

File: SimpleNameTagPlugin/pom.xml [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            <exclusions>
+                <exclusion>
+                    <groupId>com.google.guava</groupId>
+                    <artifactId>guava</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>net.md-5</groupId>
+            <artifactId>bungeecord-bootstrap</artifactId>
```
Comment (derklaro) [inline SimpleNameTagPlugin/pom.xml]: Pretty sure bungee is not required here

**Review by derklaro — CHANGES_REQUESTED** [review]
And why is the whole plugin split into one root and one sub module? There is no requirement for this...

File: SimpleNameTagPlugin/src/main/resources/plugin.yml (hunk 1 above)
Comment (TheMeinerLP) [reply SimpleNameTagPlugin/src/main/resources/plugin.yml]: KK

File: Jenkinsfile [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+      }
+    }
+    stage('Release ZIP') {
+      steps {
+        sh '''mkdir -p temp;
+        cp SimpleNameTagPlugin/target/*.jar temp/;'''
+        zip archive: true, dir: 'temp', glob: '', zipFile: 'SimpleNameTag.zip'
+        sh 'rm -r temp/';
+      }
+    }
```
Comment (GiantTreeLP) [inline Jenkinsfile]: Is a ZIP file really necessary?

File: SimpleNameTagPlugin/src/main/java/eu/cloudnetservice/simplenametag/SimpleNameTagSystem.java [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+            e.printStackTrace();
+        }
```
Comment (GiantTreeLP) [inline SimpleNameTagPlugin/src/main/java/eu/cloudnetservice/simplenametag/SimpleNameTagSystem.java:157]: Is it necessary to use reflections for 1.13+ plugins in this case?

**Review by GiantTreeLP — CHANGES_REQUESTED** [review]
Please answer the questions and improve the legibility of some parts of the code (just reformat the files manually)

File: Jenkinsfile (hunk 4 above)
Comment (TheMeinerLP) [reply Jenkinsfile]: Is not. I remove on the next commits

File: SimpleNameTagPlugin/src/main/java/eu/cloudnetservice/simplenametag/SimpleNameTagSystem.java (hunk 5 above)
Comment (TheMeinerLP) [reply SimpleNameTagPlugin/src/main/java/eu/cloudnetservice/simplenametag/SimpleNameTagSystem.java:157]: Yes is very necessary. Or we have problems with 1.13 backwards compatibility

**Pull request merged**